core/common/src/main/java/alluxio/util/JvmPauseMonitor.java

```diff
@@ -93,7 +93,7 @@ public void stop() {
     }
     mJvmMonitorThread.interrupt();
     try {
-      mJvmMonitorThread.join();
+      mJvmMonitorThread.join(60 * 1000);
```

Comment:
I don't think the jvm monitor is doing very much, so we probably don't need to wait so long. Maybe: `5 * Constants.SECOND_MS`?

Also, I think the thread while loop should be something like `while (!Thread.currentThread().isInterrupted())`